src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java

```diff
@@ -204,4 +206,17 @@ public ResponseEntity<CaseDetails> caseworkerSubmitEvent(
         }
     }
 
+    @GetMapping(path = "/{caseId}/userCaseRoles")
+    @Operation(summary = "Get user Roles for a case")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "OK"),
+        @ApiResponse(responseCode = "400", description = "Not Found"),
```

Comment:
Not found is 404

Reply:
this was my initial thought but seems like civil-service doesn't like 404 :)

![image](https://github.com/hmcts/civil-service/assets/93268657/29f2fd48-7112-4e98-9b36-3d43a0c518d0)

Reply:
hahaha just need to update the description then to bad request 😊